config.json

```diff
@@ -1,9 +1,10 @@
 {
-  "logChannel": null,
-  "action": "DELETE",
+  "token": "put your bot's token here",
+  "logChannel": "put log channel id here",
```

Comment:
I specifically set this to null so you could log into the same channel.

Reply:
Doesn't it log into the same channel if it can't resolve the channel?

Reply:
It does

![image](https://user-images.githubusercontent.com/63594377/141680385-4e5118cc-5ad6-4d42-bc38-fe6b2cad122f.png)
![image](https://user-images.githubusercontent.com/63594377/141680413-069bf1ce-55a5-477a-805c-f2e6f2640f8a.png)

Reply:
Yes, it does! That was the default behaviour in your previous case.

Reply:
Your approach also works, my was just a clean route.